hack/generator/pkg/astmodel/primitive_type.go

```diff
@@ -37,8 +37,8 @@ func (prim *PrimitiveType) AsType(codeGenerationContext *CodeGenerationContext)
 	return ast.NewIdent(prim.name)
 }
 
-func (prim *PrimitiveType) AsDeclarations(CodeGenerationContext *CodeGenerationContext, name TypeName, description *string) []ast.Decl {
-	return AsSimpleDeclarations(CodeGenerationContext, name, description, prim)
+func (prim *PrimitiveType) AsDeclarations(codeGenerationContext *CodeGenerationContext, name TypeName, description []string) []ast.Decl {
```

Comment:
I'd go further and name this something snappier like `genContext` myself. It's pretty clear from the context (heh) what this is.

Reply:
Good idea.